fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/functions/Converter.java

```diff
@@ -24,17 +24,29 @@
  * @param <A> the type from which we are translating
  * @param <B> the type to which we are translating
  */
-public interface Converter<A, B> extends InvertibleFunction<A, B>, DomainRestrictedFunction<A, B> {
+public interface Converter<A, B> extends InjectiveFunction<A, B>, DomainRestrictedFunction<A, B> {
+
+    @Override
+    public default Converter<B, A> inverse() {
```

Comment:
If you want to put `toDomain` in, take this out, or have them both but in a subtype of `Converter`, same style as `InjectiveFunction` < `InvertibleFunction`.